packages/flutter_driver/lib/src/driver/driver.dart

```diff
@@ -483,7 +483,7 @@ class FlutterDriver {
     // the chance to complete if the item is already onscreen; if not, scroll
     // repeatedly until we either find the item or time out.
     bool isVisible = false;
-    waitFor(item, timeout: timeout).then((Null _) { isVisible = true; });
+    waitFor(item, timeout: timeout).then((Null value) { isVisible = true; });
```

Comment:
Just curious: we stopped using underscore for unused parameters?

Reply:
Yeah at some point I realised that using `_` reduces the self-documenting aspect of the code. Not that "value" is much better than "_" but at least it's clear that it's not something like "time" or "error".